### DISCUSSION TITLE
Multi port usage

### DISCUSSION THREAD

**Discussion by martinribelotta** [cherry-embedded/CherryUSB#72] [General]
I'm using cherryUSB with ch32v307 and stm32h750... both have two usb interfaces supported by this stack, but I'm not sure how to use both in simultaneos... 

Maybe require code/interface modification?

**Comment by sakumisu** [comment]
This need consider and will be implemented for a long time ,all api need to add one param to match this.

**Reply by sakumisu** [reply]
This work is going on.

**Reply by sakumisu** [reply]
Done, please checkout branch multiport-dev
